src/eth/EthereumCdpService.js

```diff
@@ -105,8 +106,13 @@ export default class EthereumCdpService extends PrivateService {
     });
   }
 
-  async lockEth(cdpId, amount, unit = ETH) {
-    await this._conversionService().convertEthToWeth(amount, unit);
+  @tracksTransactions
+  async lockEth(cdpId, amount, { unit = ETH, promise }) {
```

Comment:
note that the `unit` argument has been moved into the options -- this is to avoid the pain of having two optional arguments